### PR TITLE
Add Scoop to readme, issue template and label workflow

### PR DESCRIPTION
**Pull request by efb4f5ff-1298-471a-8973-3d47447115dc**
---
Add Scoop to readme
---

**Pull Request Type**
Please select what type of pull request this is:
- [ ] Bugfix
- [ ] Feature Implementation
- [x] Documentation
- [ ] Other

**Related issue**
https://github.com/FreeTubeApp/FreeTube/discussions/2539

### CODE REVIEW

**Review by absidue — APPROVED** [review]

**Comment by ChunkyProgrammer** [comment]
ReadMe is getting pretty big. I think we should have links at the top of it to navigate to the different headings. Ex: https://github.com/FreeTubeApp/FreeTube#localization

**Review by ChunkyProgrammer — APPROVED** [review]

File: .github/workflows/autoLabelIssue.yaml [hunk 1]
```diff
@@ -10,5 +10,5 @@ jobs:
...
... ["usability issue"], "labels": ["B: usability"]}, {"keywords": ["causes crash"], "labels": ["B: crash"]}, {"keywords": ["feature stopped working"], "labels": ["B: feature stopped working"]}, {"keywords": ["inconsistent behavior"], "labels": ["B: inconsistent behavior"]}, {"keywords": ["data loss"], "labels": ["B: data loss"]}, {"keywords": ["race condition"], "labels": ["B: race condition"]}, {"keywords": ["API issue"], "labels": ["B: API issue"]}, {"keywords": ["only happens in developer mode"], "labels": ["B: developer mode"]}, {"keywords": ["improvement to existing feature"], "labels": ["E: improvement existing feature"]}, {"keywords": ["new optional setting"], "labels": ["E: new optional setting"]}, {"keywords": ["visual improvement"], "labels": ["E: visual improvement"]}, {"keywords": ["display more information to user"], "labels": ["E: display more information"]}, {"keywords": ["ease of use improvement"], "labels": ["E: ease of use improvement"]}, {"keywords": ["support for external software"], "labels": ["E: support external software"]}, {"keywords": ["new feature"], "labels": ["E: new feature"]}, {"keywords": ["new keyboard shortcut"], "labels": ["E: keyboard shortcut"]}]'
```
Comment (ChunkyProgrammer) [inline .github/workflows/autoLabelIssue.yaml:13]: any idea if we could make this multilined?

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/workflows/autoLabelIssue.yaml:13]: I tried allot of things a few months back but all didn't work out. I'll try to investigate more to make it more readable

**Review by PikachuEXE — APPROVED** [review]

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> ReadMe is getting pretty big. I think we should have links at the top of it to navigate to the different headings. Ex: https://github.com/FreeTubeApp/FreeTube#localization (this is for a different pr)

@ChunkyProgrammer Is this necessary? I just found out we have this

https://user-images.githubusercontent.com/73130443/189497312-a065e725-0514-4833-85ab-2883f3b939dc.mp4

**Pull request merged**